VM/plot_vm.py

```diff
@@ -24,32 +60,50 @@ def plot_vm(
 
     Parameters
     ----------
-    vm_params_dict :
-    srf_corners :
-    land_outline_path :
-    centre_line_path :
-    mag :
-    outdir :
-    ptemp :
-    logger :
+    vm_params_dict : dict
+        Dictionary extracted from vm_params.yaml file.
+    srf_corners : np.ndarray.
+        Corners of SRF planes. Formatted as [[[lon,lat],[lon,lat],[lon,lat],[lon,lat]],...]
+    land_outline_path : Path
+        Path to the land outline file
+    centre_line_path : Path
+        Path to the centre line file
+    mag : float
+        Magnitude of the event
+    outdir : Path
+        Output directory
+    ptemp : Path
+        Temporary directory
+    logger : Logger
+        Default is qclogging.get_basic_logger()
     """
 
+    from rel2vm_params import write_srf_path
+
     logger.debug("Plotting vm")
     p = gmt.GMTPlot(ptemp / "optimisation.ps")
     p.spacial("M", vm_params_dict["plot_region"], sizing=7)
     p.coastlines()
 
-    srf_path = ptemp / "srf.path"
-    if srf_path.exists():
+    # SRF domain
+    if srf_corners is not None and len(srf_corners) > 0:
```

Comment:
can be summarised as `if srf_corners:`

Reply:
`if srf_corners:`  causes an exception:
```
Traceback (most recent call last):
  File "/tmp/Pre-processing/VM/plot_vm.py", line 245, in main
    plot_vm(
  File "/tmp/Pre-processing/VM/plot_vm.py", line 85, in plot_vm
    if srf_corners:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
python-BaseException
```
As srf_corners is np.ndarray, I can happily drop the None-check, but keep `len(srf_corners) > 0`